### PR TITLE
Bring PDF viz support to Julia

### PR DESCRIPTION
**Pull request by jeremiahpslewis**
Enable vector plot output for quarto PDF docs for Julia libraries CairoMakie.jl and Plots.jl

### CODE REVIEW

**Comment by jeremiahpslewis** [comment]
- Tested locally, looks reasonable for Plots and CairoMakie.
- Shifted from `using Plots` to `import Plots` to handle namespace conflicts, user can add a `using Plots` as needed

File: src/resources/jupyter/lang/julia/setup.jl [hunk 1]
```diff
@@ -10,30 +10,49 @@ import IJulia
...
+fig_width = {0}
+fig_height = {1}
+fig_format = :{2}
+fig_dpi = {3}
+
+# no retina format type, use svg for high quality type/marks
+if fig_format == :retina
+  fig_format = :svg
+elseif fig_format == :pdf
+  fig_dpi = 96
```
Comment (jjallaire) [inline src/resources/jupyter/lang/julia/setup.jl:22]: Do we want the DPI to be left alone here? (we default it to 300 for PDF, we defaulted it back to 96 only b/c we were re-routing to PNG)

Comment (jjallaire) [reply src/resources/jupyter/lang/julia/setup.jl:22]: I see now that we need the dpi to be 96 for the pixel calculation to be correct

**Comment by jjallaire** [comment]
I'll go ahead and merge this notwithstanding the comments about dpi (I'll assume you'll come back with a different PR if you think some adjustment in how we handle dpi for PDF is warranted)

**Comment by jjallaire** [comment]
Hi there @jeremiahpslewis, posting back here because I'm giving a talk on Quarto at JuliaCon next month and was hoping to find some good examples of using Julia w/ Quarto. If you are able to share any of your own (or are aware of others) please LMK over on this discussion thread: https://github.com/quarto-dev/quarto-cli/discussions/1261. Thanks!

**Pull request merged**